Tchap/Modules/Home/HomeCoordinator.swift

```diff
@@ -121,6 +121,14 @@ final class HomeCoordinator: NSObject, HomeCoordinatorType {
         self.navigationRouter.present(publicRoomsCoordinator, animated: true)
         publicRoomsCoordinator.delegate = self
     }
+    
+    private func startDiscussion(with userID: String) {
```

Comment:
Please add a comment to highlight that no discussion (room) is existing yet, we open an empty room view controller to prepare it.
I would suggest rename this method with "prepareDiscussion"